k8s/helmfile/helmfile.yaml

```diff
@@ -152,7 +152,7 @@ releases:
 
   - name: api
     chart: wbstack/api
-    version: "0.31.0"
+    version: "0.31.1"
```

Comment:
This is a noop as long as the serviceMonitor is not explicitly enabled, so it's fine to also ship this to production for now.